### ISSUE TITLE
`blocks()` does not work with Rmd files

### ISSUE THREAD

**Issue by billdenney** [metrumresearchgroup/mrgsolve#588]
I'm guessing that this is a minor issue of how parsing occurs:

``` r
library(mrgsolve)
#> 
#> Attaching package: 'mrgsolve'
#> The following object is masked from 'package:stats':
#> 
#>     filter
rmd_file <- modlib("popex.Rmd")
#> Building popex_Rmd ...
#> done.
cpp_file <- modlib("popex.cpp")
#> Building popex_cpp ... done.
blocks(rmd_file)
#> Error: No model specification file blocks were found.
blocks(cpp_file)
#> 
#> Model file: popex.cpp 
#> 
#> $PARAM
#> TVKA = 0.5, TVCL = 1, TVV = 24
#> 
#> $PARAM
#> @covariates
#> WT = 70
#> 
#> $MAIN
#> double CL = exp(log(TVCL) + 0.75*log(WT/70) + ECL);
#> double V  = exp(log(TVV)  +      log(WT/70) + EV );
#> double KA = exp(log(TVKA)                   + EKA);
#> 
#> $TABLE
#> capture IPRED = CENT/V;
#> capture DV = IPRED*exp(EPS(1));
```

<sup>Created on 2019-11-20 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by kylebaron** [comment]
Yeah ... it doesn't look for Rmd extension and never picks `modelparse_rmd` to take the model apart.

**Comment by kylebaron** [comment]
Closing this;  see issue #608 that will be referenced in the SDLC docs

**Comment by kylebaron** [comment]
This is fixed in 0.10.1  Thanks, @billdenney

**Comment by billdenney** [comment]
Thanks!
